deployment/ingress/eks/ingress.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: ${k8s_namespace}-ingress
+  namespace: ${k8s_namespace}
+  annotations:
+    nginx.ingress.kubernetes.io/ssl-redirect: "false"
+    nginx.ingress.kubernetes.io/force-ssl-redirect: "false"
+    nginx.ingress.kubernetes.io/rewrite-target: /
+    cert-manager.io/cluster-issuer: "letsencrypt-prod" 
+spec:
+  rules:
+    - host: ${ingress_dns}
+      http:
+        paths:
+          - path: /
+            pathType: Prefix
+            backend:
+              service:
+                name: ${k8s_namespace}-service
+                port:
+                  number: ${ingress_http_port}
+  tls:
+    - hosts:
+        - ${ingress_dns}
+      secretName: letsencrypt-prod
```

Comment:
We should have docs or scripts for configuring this secret

Reply:
I can add a doc, but the secret is created automatically from ClusterIssuer , see

https://cert-manager.io/docs/configuration/acme/

Simply the deploy ingress script will take care of this -> I need to add instructions on it though